go.mod

```diff
@@ -71,3 +71,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/meroxa/meroxa-go => ../meroxa-go
```

Comment:
looks good, will wait for you to update meroxa-go and drop this before approving ✅